### PR TITLE
Changelog Rust 1.48

### PR DESCRIPTION
**Pull request by flip1995**
[Rendered](https://github.com/flip1995/rust-clippy/blob/changelog/CHANGELOG.md)

This changelog is **big**. Hacktoberfest was pretty good for us, even though we had too few reviewers.

changelog: none

### CODE REVIEW

**Comment by flip1995** [comment]
Please do not merge yet, because I'll merge it when I'm done with the [Clippy release process](https://github.com/rust-lang/rust-clippy/blob/master/doc/release.md#update-changelogmd) in the last step.

**Review by Manishearth — APPROVED** [review]
wooo!

**Pull request merged**
